File: app/src/main/java/com/introduce/hotel/model/WeatherResponse.java
package com.introduce.hotel.model;
import com.google.gson.annotations.SerializedName;

public class WeatherResponse {
    @SerializedName("temperature_2m")
    private float temperature;
    @SerializedName("relative_humidity_2m")
    private float humidity;
    @SerializedName("wind_speed_10m")
    private float windSpeed;

    public float getTemperature() {
        return temperature;
    }

    public float getHumidity() {
        return humidity;
    }

    public float getWindSpeed() {
        return windSpeed;
    }
}

File: app/src/main/java/com/introduce/hotel/model/WeatherResponse.kt
package com.introduce.hotel.model

import com.google.gson.annotations.SerializedName

data class WeatherResponse(
        @SerializedName("current") val current: Current
) {
    data class Current(
            @SerializedName("time") val time: String,
            @SerializedName("temperature_2m") val temperature_2m: Double,
            @SerializedName("wind_speed_10m") val wind_speed_10m: Double
    )
}
